state/protocol/epoch.go

```diff
@@ -14,6 +14,9 @@ type EpochQuery interface {
 
 	// Next returns the next epoch as of this snapshot. Valid snapshots must
 	// have a next epoch available after the transition to epoch setup phase.
+	//
+	// Returns ErrNextEpochNotSetup in the case that this method is queried w.r.t.
```

Comment:
Can we mention that we get an invalid epoch which will return exact sentinel. I feel like it's confusing for people who haven't worked with the internals.